portal/templates/redesign/teach_new/materials_new.html

```diff
@@ -245,25 +246,26 @@ <h1 class="banner__text--primary">Teaching Resources</h1>
             <h4>Children will learn the basics of coding by programming a delivery van to drive around simple routes. They will sequence instructions and use simple repetition
                 commands in a visual programming language.</h4>
             <a href="{{ 'KS2.zip'|cloud_storage }}" class="button button--regular button--primary--setup">Download all Lower KS2 resources</a>
+            <img title="ZIP" alt="ZIP icon" src='{% static "portal/img_new/zip.png" %}'>
         </section>
-        <h3>Sessions</h3>
+        <h3>Sessions <span class="ks-tag">Lower KS2</span></h3>
         <p class="body-text">Complete lesson plans for each session, including learning objectives, a list of useful resources, pupil activities, and guideance on share and
             review following the session.</p>
         <table class="table--resources">
             <tr>
                 <td><p class="body-text"><strong>Session 1</strong></p></td>
                 <td><p class="body-text">
-                    <a href="{% url 'materials_viewer_new' pdf_name='lks2_session_1' %}">Recap on using a simple 'repeat' loop</a></p></td>
+                    <a href="{% url 'materials_viewer_new' pdf_name='lks2_session_1' %}">Recap on using a simple &lsquo;repeat&rsquo; loop</a></p></td>
             </tr>
             <tr>
                 <td><p class="body-text"><strong>Session 2</strong></p></td>
                 <td><p class="body-text">
-                    <a href="{% url 'materials_viewer_new' pdf_name='lks2_session_2' %}">Using repeat with a condition on a simple route: 'repeat until'</a></p></td>
+                    <a href="{% url 'materials_viewer_new' pdf_name='lks2_session_2' %}">Using repeat with a condition on a simple route: &lsquo;repeat until&rsquo;</a></p></td>
             </tr>
             <tr>
                 <td><p class="body-text"><strong>Session 3</strong></p></td>
                 <td><p class="body-text">
-                    <a href="{% url 'materials_viewer_new' pdf_name='lks2_session_3' %}">Using simple selections and repetition: Repeat until at destination 'if…do'</a>
+                    <a href="{% url 'materials_viewer_new' pdf_name='lks2_session_3' %}">Using simple selections and repetition: Repeat until at destination &lsquo;if…do&rsquo;</a>
```

Comment:
inconsistent with how you previously write 'repeat' loops (small letters, quotes)